src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -37,6 +37,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Dim operation = TryCast(node, IOperation)
                 If operation IsNot Nothing Then
                     Me.nodes.Add(operation)
```

Comment:
We seem to have adopted a convention of naming fields with a leading underscore. This field is not introduced in this change set, though.